collector/processor/concurrentbatchprocessor/config.go

```diff
@@ -68,8 +68,5 @@ func (cfg *Config) Validate() error {
 	if cfg.Timeout < 0 {
 		return errors.New("timeout must be greater or equal to 0")
 	}
-	if cfg.MaxInFlightSizeMiB <= 0 {
```

Comment:
Should we still validate cfg.MaxInFlightSizeMiB can't be negative?

Reply:
The field is unsigned, so there's no such test.